graphql/resolvers/alumnis.js

```diff
@@ -118,5 +121,116 @@ module.exports = {
 
       return newAlumni;
     },
+
+    async editAlumniProfile(
+      _,
+      {
+        editAlumniProfileInput: {
+          firstName,
+          lastName,
+          oldEmail,
+          newEmail,
+          undergrad,
+          grad,
+          employer,
+          position,
+          location,
+          linkedin,
+        },
+      }
+    ) {
+      const { errors, valid } = validateEditAlumniProfileInput(
+          firstName,
+          lastName,
+          oldEmail,
+          newEmail,
+          undergrad,
+          grad,
+          employer,
+          position,
+          location,
+          linkedin,
+      );
+
+      var coordinates = {
+        latitude: 0,
+        longitude: 0
+      };
+
+      if (!valid) {
+        handleInputError(errors);
+      }
+
+      grad.year = grad.year === "" ? 0 : grad.year;
```

Comment:
give [this](https://frontend.turing.edu/lessons/module-1/js-truthy-falsy-expressions.html#:~:text=Truthy%20values%20In%20JavaScript%2C%20a,encountered%20in%20a%20Boolean%20context.) article a read. In JS avoid writing statements like x === null or x === 0 or x === "".

Reply:
The schema calls for an integer, so if the user leaves the field empty, it needs to be converted from a string to an integer.